code_strategy.md

```diff
@@ -0,0 +1,112 @@
+We are biomedical data scientists who use code to answer biological hypotheses and explore biological data. It is therefore important to define how we use, write, and share code. 
+
+## 1. Using code
+
+Code is ubiquitous on the internet, and there are many examples of both elegant and useless code. We leverage all possible resources to achieve our goals; using both trusted software and validated code snippets where available.
+
+We use trusted software wherever possible, selecting to use code that is tried and true, with a strong, active community. When we use these software packages, we specifically take note of which version we’re using, as different versions of different software have different functionality, and often induce different results.
```

Comment:
Recommend updating the second sentence here. The use of `different` is a bit too repetitive in my opinion.